packages/core/src/lib/markdown-it/plugins/markdown-it-icons.ts

```diff
@@ -7,8 +7,13 @@ function getOcticonIcon(iconName: string) {
   return octicons[iconName as IconName] ?? null;
 }
 
+const ICON_REGEXP = new RegExp(
```

Comment:
We should be using literal as per https://github.com/MarkBind/markbind/pull/1956
...but strange that eslint is not calling this out